tests/boards/tenstorrent/tt_blackhole/dmc/gpiox/src/main.c

```diff
@@ -13,21 +13,26 @@
 #define NGPIOS 8
 
 static const struct device *const ports[] = {
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox0)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox1)),
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox2)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox3)),
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox4)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox5)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox2)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox3)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox4)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox5)),
```

Comment:
chip0_strapping and chip1_strapping don't seem to be present on p100, but I guess it makes sense to add them to the end of the list.

The other GPIO expanders should be fine

```suggestion
	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox0)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox1)),
	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox2)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox3)),
	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox4)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox5)),
```